spec/GraphHopperMatrixSpec.js

```diff
@@ -0,0 +1,20 @@
+var GraphHopperMatrix = require('../src/GraphHopperMatrix');
+var GHInput = require('../src/GHInput');
+var ghMatrix = new GraphHopperMatrix({key: key, vehicle: profile});
+
+
+describe("Matrix Test", function () {
+    it("Get results", function (done) {
+        ghMatrix.addOutArray("distances");
+        ghMatrix.addPoint(new GHInput("52.651395,13.15567"));
+        ghMatrix.addPoint(new GHInput("52.432572,13.143539"));
+        ghMatrix.addPoint(new GHInput("52.43299,13.461571"));
+        ghMatrix.addPoint(new GHInput("52.622226,13.381233"));
+
+        ghMatrix.doRequest(function (json) {
+            expect(json.message).not.toBeDefined();
+            expect(json.distances).not.toBeLessThan(0);
```

Comment:
As distances is an array why will it pass? and I would also like to have a rough estimate like distance should be at least 5km but not more than 10km or something. WDYT?